src/main/java/techcourse/myblog/domain/CommentRepository.java

```diff
@@ -5,5 +5,5 @@
 import java.util.List;
 
 public interface CommentRepository extends JpaRepository<Comment, Long> {
-    List<Comment> findByArticle(Article article);
+    List<Comment> findByArticleId(Long articleId);
```

Comment:
메서드 이름은 메서드의 정확한 동작을 구체적으로 나타내는 것이 좋겠죠?

```suggestion
    List<Comment> findAllByArticleId(Long articleId);
```